src/fuzz_introspector/analysis.py

```diff
@@ -183,14 +183,12 @@ def get_node_coverage_hitcount(
         # As this is the first node ensure it is indeed the entrypoint.
         # The difference is this node has node "parent" or prior nodes.
 
-        # Arthur: Not true anymore for cpp or jvm with multiple yaml file
-        # TODO Relaxing this constraint temporary, fixed later
-        # if not profile.func_is_entrypoint(demangled_name):
-        #     raise AnalysisError(
-        #         "First node in calltree seems to be non-fuzzer function"
-        #     )
-
+        if not profile.func_is_entrypoint(demangled_name):
```

Comment:
This one we should watch out for as I'm not sure the condition will always be true. I'm thinking with regards to Python here.